Documentation/platforms.md

```diff
@@ -0,0 +1,85 @@
+# Target Platforms
+
+MonoGame supports the following systems:
+
+- Windows
+- Mac
+- Linux
+- Android
+- iOS
+- PlayStation 4
+- PlayStation Vita
+- Xbox One
+- Nintendo Switch
+- Google Stadia
+
+There are different implementations of MonoGame that we call target platforms (or just platforms).
+The platforms mostly correspond to the systems MonoGame supports but some platforms
+support multiple systems. Platforms for systems that require registration such as PlayStation
+and Xbox are not publicly available. To get access to those platforms, contact Tom Spilman
+at ???. MonoGame documentation for closed platforms is available in their respective repositories.
+
+Below is a list of public platforms with the corresponding NuGet package, the `dotnet new` template identifier
+and an explanation of the platform.
+
+- [WindowsDX](#windowsdx)
+- [DesktopGL](#desktopgl)
+- [Windows Universal](#windowsuniversal)
+- [Android](#android)
+- [iOS](#ios)
+
+## WindowsDX
+
+**Supported Systems**: Windows
+**NuGet Package**: MonoGame.Framework.WindowsDX
+**Template ID**: mgwindowdx
+
+WindowsDX uses WinForms to manage the game window. DirectX (9.0c or newer) is used for graphics
+and XAudio is used for audio. You can target Windows Vista or up with this platform.
+
+## DesktopGL
+
+**Supported Systems**: Windows, Mac, Linux
+**NuGet Package**: MonoGame.Framework.DesktopGL
+**Template ID**: mgdesktopgl
+
+DesktopGL uses SDL for windowing. OpenGL is used for graphics and
+OpenAL Soft for audio. DesktopGL supports Windows (Vista or up),
+Mac (version ???) and Linux.
```

Comment:
@mrhelmut I don't know what the minimum macOS version we support is

Reply:
From the API perspective, none specifically.
.Net Core however, has a High Sierra (macOS 10.13) requirement as of .Net Core 3.